src/coreclr/jit/emitarm64.cpp

```diff
@@ -9924,6 +9952,12 @@ void emitter::emitIns_R_R_R(instruction     ins,
             fmt = IF_SVE_CZ_4A;
             break;
 
+        case INS_sve_ldnf1b:
+        case INS_sve_ldnf1h:
+        case INS_sve_ldnf1w:
+        case INS_sve_ldnf1d:
+            return emitIns_R_R_R_I(ins, size, reg1, reg2, reg3, 0, opt);
```

Comment:
this doesn't look right. The caller should make sure to call appropriate `emitIns*` method.